plugins/chunter-resources/src/components/Message.svelte

```diff
@@ -59,6 +60,23 @@
         action: chunter.actionImpl.SubscribeMessage
       } as Action)
 
+  async function deleteMessage () {
+    if (space) {
+      await client.removeDoc(chunter.class.Message, space, message._id)
```

Comment:
You have space in message, you don't need it. Also you should use `await client.remove(message)`

Reply:
Done